IdentityCore/src/webview/embeddedWebview/MSIDOAuth2EmbeddedWebviewController.m

```diff
@@ -315,6 +315,20 @@ - (void)webView:(WKWebView *)webView didReceiveAuthenticationChallenge:(NSURLAut
                         completionHandler:completionHandler];
 }
 
+- (void)webView:(WKWebView *)webView decidePolicyForNavigationResponse:(WKNavigationResponse *)navigationResponse decisionHandler:(void (^)(WKNavigationResponsePolicy))decisionHandler
+{
+    if (_navigationResponseBlock && navigationResponse && navigationResponse.response)
+    {
+        NSHTTPURLResponse *response = (NSHTTPURLResponse *)navigationResponse.response;
+        if (response)
+        {
+            self.navigationResponseBlock(response);
```

Comment:
You're using "self.navigationResponseBlock" and "_navigationResponseBlock" above. Can you update to consistently use self.navigationResponseBlock?

Reply:
I got a warning when I first added it as `self.navigationResponseBlock`, but I no longer see it (and don't remember what was the previous warning). Updated.